---
layout: post
title: "exceRpt Tool: Publication in Cell Systems"
author: James Diao
date: 2019-04-04
location: New Haven, CT
categories: Research
excerpt: Publication of my first computational research project (quality control analysis and filtering for exRNA-Seq samples)
images:
  - url: /img/0-excerpt.png
---


### tl;dr 
The first computational research project I worked on in [Gerstein Lab](/research/gerstein-lab) (debugging and quality control analysis for an exRNA-Seq analysis pipeline) was finally published, nearly 4 years after I started working on it. 

> Rozowsky J, Kitchen RR, Park JJ, Galeev T, **Diao JA**, Warrell J, Thistlethwaite W, Subramanian SL, Milosavljevic A, Gerstein M. "exceRpt: a comprehensive analytic platform for extracellular RNA profiling." *Cell Systems*. 4 Apr 2019. [https://doi.org/10.1016/j.cels.2019.03.004](https://www.cell.com/cell-systems/fulltext/S2405-4712(19)30074-2).

<img src="/img/excerpt-title.png" alt="exceRpt Header" title="exceRpt Header" style="width:700px;height:284px;">  

### Non-technical summary

Our bodily fluids (e.g., blood, urine, etc.) contain many different molecules that can be used to diagnose and track disease. Most non-invasive diagnostic tests--from pregnancy tests to cancer biomarkers--look for abnormal levels of specific proteins. More recently, scientists have discovered that another type of molecule, exRNA, can also be measured from fluid samples for diagnostic purposes. To help standardize the study of exRNA, the NIH put together a consortium--the Extracellular RNA Communication Consortium (ERCC). 

The analysis of exRNA presents a number of technical problems. For example: there's not that much of it floating around in most bodily fluids, and sequencing samples are often vulnerable to contamination and artifacts. To address these challenges, our lab developed a pipeline called **exceRpt**: **ex**tra-**ce**llular **R**NA **p**rocessing **t**oolkit. exceRpt consists of a series of filtering steps (to remove contamination and artifacts) and alignment steps (to match observed molecular sequences with known sequences). It accepts raw sequence reads as inputs and returns abundance estimates and quality control reports (I contributed to this part!) as outputs. 

The hope is that everyone can process their sequencing data using the same pipeline, so that their outputs are all comparable and fully reproducible. So far, exceRpt has been used to process thousands of exRNA-seq datasets in the public exRNA Atlas (our [paper](/research/exrna-atlas) on this came out today too!) and is freely available at [http://genboree.org/site/exrna_toolset](https://genboree.org/site/exrna_toolset/) and [github.gersteinlab.org/exceRpt](https://github.gersteinlab.org/exceRpt).


### Reflection 

It's been interesting to reflect on how much has happened throughout the course of this project. Just looking back on our 2017 and 2018 [abstracts in ISEV](/research/isev-abstracts) is enough to show how far the research has come. At the same time, I also learned a lot about the process, pace, and zeitgeist of bioinformatics research. 

I started working on exceRpt as a freshman in undergrad--I had no experience with sequencing analysis, no experience working with computing clusters, and no experience with R, Python, or UNIX. It took quite a while before I was able to do anything useful. But with the help of my very patient mentor and a whole lot of practice, I was able to make something of my summer. Over those 2 months, I ran a few hundred human and mouse sequence files from the public Sequence Read Archive (SRA) through a preliminary version of the exceRpt pipeline. In the process, I worked on testing and debugging errors to make the pipeline more robust to malformed inputs. I also worked on testing and visualizing quality control metrics, including the transcriptome-mapped reads and transcriptome-to-genome mapped reads ratio metrics that ended up in Figure 2D. 

It was a wonderful learning experience, and I continued to expand on this work over the school year. Finally, when the first papers published using this pipeline started to emerge, my mentor told me that the lab was going to publish a manuscript on the pipeline, and that I could contribute to its writing as a co-author. At that point, I had already set my heart on exploring more clinical informatics research, but it was wonderfully exciting news, and I eagerly kept tabs on the progress of the manuscript. 

Over the next few years, various issues came up. Rob (the original first-author) left the lab to join a biotech startup. His new job kept him busy and the momentum to publish stalled. On our end, other ERCC projects began to take priority, and the manuscript remained incomplete for years. Finally, in late 2017 (now 2.5 years into the project), my mentor wrote up the paper and began revisions among our co-authors. 

But even with the completed manuscript, the lab was busy working on multiple projects with the ERCC. These were intended to be submitted as a "publication package," where 20 or so papers would be submitted simultaneously to the same journal and its subjournals; the  intention was that all of these highly related findings and discoveries could be referenced in the same place. The full package of 18 papers, when finally published, looked like [this](https://www.cell.com/consortium/exRNA):

<img src="/img/ercc-package.png" alt="ERCC Publication Package" title="ERCC Publication Package" style="width:700px;height:375px;">  

Coordinating preparation for nearly 20 papers by hundreds of co-authors across the multi-institutional consortium proved to be challenging. Peer review (and the additional work requested by reviewers) added even more time. By the time the papers came out, it had been almost exactly 4 years since I started everything. 

### Concluding thoughts

Although this path to publication was rocky and long (and I had definitely given up on it multiple times), I'm grateful that everything ended up working out. I know many friends whose mentors were less supportive or less willing to recognize them for their work, or who happened to encounter more unreasonable reviewers. I'm proud of the work that went into the pipeline and I think it's a really good thing for science and reproducibility that the full details of the ERCC's processing pipeline are peer-reviewed and widely distributed. 

I've since moved on from molecular bioinformatics work, but this project was an invaluable first step into computational research and data analysis, and I'm very glad that we were able to bring this work to a clean conclusion.  


